src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEPropertySymbol.cs

```diff
@@ -93,7 +93,6 @@ private enum Flags : byte
             PropertyDefinitionHandle handle,
             PEMethodSymbol getMethod,
             PEMethodSymbol setMethod,
-            int countOfCustomModifiers,
```

Comment:
>int countOfCustomModifiers, [](start = 12, length = 27)

This change is good. However, it doesn't look related to the goal of the PR or to the init-only properties feature in general. I think it would be more appropriate to separate it out into a different PR and submit it to master, so that we could benefit from it immediately without waiting for this feature branch to get integrated into master, which is likely to take weeks, if not months.